style.css

```diff
@@ -0,0 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+/*
+        * this style is my primary style
+            it will include
+            primary color,
+            d-noen,
+            d-flex,
+            justify-content-center,
+            justify-content-between,
+            justify-content-around,
+            weight--700,
+            weight--400,
+            weight--500,
+            align-items-center,
+            align-self-center,
+            align-content-center
+        */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.mt-12 {
+  margin-top: 12px;
+}
+
+.weight--400 {
+  font-weight: 400;
+}
+
+.weight--500 {
+  font-weight: 500;
+}
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. The typo, color, font, and spacing are supposed to be similar to the original design provided in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205046045-72a2c1fc-e8ce-480b-92c5-1a58ccaafae5.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205046154-d8a080ce-9f3e-40e3-9544-294670c37c44.png)